### DISCUSSION TITLE
Problems after moving Remote Apps to Win10 Host

### DISCUSSION THREAD

**Discussion by Rincewindwiz** [kimmknight/remoteapptool#115] [Q&A]
Hi
I'm far from a Windows Wizz having learned about computers back in the late 60s! But I managed to get a VFP program that manages scoring in a large amateur event set up under Remote apps and have used it successfully for several years. Absolutely fine. No problems. Thank you.

The host PC is a little long in the tooth so I have moved it to a Win10 PC (using the same RDP files as before) but it does nto work nearly as slickly.

Previously (Win7 host) the user would access the app via RAwebb. The app would fire up in its own small window, work fine and when the user exited the app the window would close and the user logged off the host machine. 

On the Win10 host,
While the app logs onto windows the user sees a blurred version of the log on screen - previously the logon process was invisible.
After log on the user sees the windows home screen + short cuts to the app - previously the remote app fired up in its own window
When the user runs the app he sees the fil windows screen with the app running in its own window - previously only the app's window was visible
When he exits the app he also has to log off windows - previously he was also logged off windows.

All of this is a serious step back from the Win7 host so how can I get back to the nice slick operation that existed under Win 7?

Thanks for any info

PS: I did not reinstall any of the remote app software (other than of course asp to get the inetpub directory and contents) just copied the RDP files across

**Comment by MrBrianGale** [comment]
I think you answered your own question there - you didn't install the remote app software on the server. IF the server doesn't have remote apps configured on it, the RDP connection can't "start" the remote app. The server needs to have remote app software installed and configured on it to allow the remote app to start.
Unless I am misunderstanding something with how you set stuff up that is, but the way I read this is that you replaced the server which used to be Win7 with a Win10 box and the client machines can't get remote apps to work now. If this is the case, you will need to set up the remote apps again on the Win10 box. Replacing the server means you have to set everything up again. Replacing the clients means you can just copy the RDP file over.

**Reply by Rincewindwiz** [reply]
I didn't install (usual lexcuse - too little time to do it properly!) and the next step is to do just that BUT
The clients do connect to the remote apps on the Win10 server (if the didn't software install would have been the obvious next step) - its just not as slick (see description above) as it was with the WIN 7 server.

I'll have a go at the install and see if that improves things

**Reply by MrBrianGale** [reply]
If a remote app server isn't installed on the server, how are the clients going to connect to it?
The RDP file just tells the clients the server name and some connection information (such as application to try to load). The SERVER MUST have a remote app server set up for the clients to work.
Once you have the Win 10 server configured the same as the Win 7 server, the behavior should be identical unless something has changed client side as well.
